Mathlib/Combinatorics/SimpleGraph/Subgraph.lean

```diff
@@ -644,6 +644,9 @@ lemma map_monotone : Monotone (Subgraph.map f) := fun _ _ ↦ map_mono
 theorem map_sup (f : G →g G') (H₁ H₂ : G.Subgraph) : (H₁ ⊔ H₂).map f = H₁.map f ⊔ H₂.map f := by
   ext <;> simp [Set.image_union, map_adj, sup_adj, Relation.Map, or_and_right, exists_or]
 
+@[simp] lemma map_equiv_top {H : SimpleGraph W} (e : G ≃g H) : Subgraph.map e.toHom ⊤ = ⊤ := by
```

Comment:
"equiv" doesn't seem the right name for this given that it doesn't contain an "equiv", instead an iso

Reply:
Indeed